### ISSUE TITLE
[Question] Scope for string interpolation

### ISSUE THREAD

**Issue by iaduran** [wesbos/cobalt2#150]
Hi and thanks for a great theme.

I have a question. This is a screenshot of your JavaScript30 course, second video:

![screen shot 2017-06-02 at 11 22 09 pm](https://cloud.githubusercontent.com/assets/20104703/26743443/810eacb2-47ea-11e7-85ff-9fcbb87cf3f2.png)

and this is a screenshot from my Sublime (Build 3126) with default JavaScript syntax:

![screen shot 2017-06-02 at 11 25 14 pm](https://cloud.githubusercontent.com/assets/20104703/26743492/c5e00458-47ea-11e7-8483-762d0b949b7d.png)

I managed to add the same foreground color to `querySelector` and `play` by adding the `variable.function` scope to the theme. However, I haven't managed to get the same color for the `keyCode` attribute inside the string interpolation. Mine is green while yours is orange or yellow. Also, the `backticks` in yours are white while mine are green. 

So basically I'm wondering which scope targets `keyCode` attribute inside the string interpolation, and also the color of the backticks.

Thanks in advance!
